apps/src/templates/ToggleGroup.jsx

```diff
@@ -26,15 +26,15 @@ const ToggleGroup = React.createClass({
       });
       return error;
     }
-  },
+  };
 
   setSelected(selected) {
```

Comment:
ideally, this would also be updated to an arrow method so it doesn't need to be bound in `ToggleButton.onClick`, and either `ToggleButton` itself or a new parent component should be taught how to pass the `value` to the click handler (see [the eslint docs](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md#lists-of-items)), but that might be out of scope for this PR

Reply:
Good catch.  I'll address in a follow up when I convert `ToggleButton` to ES6.